native/core/src/models.rs

```diff
@@ -59,10 +65,12 @@ use crate::{
 
 #[derive(Debug, Eq, PartialEq)]
 pub enum Capability {
-    // someone can comment on this
+    // someone can add reaction on this
+    Reactable,
+    // someone can add comment on this
     Commentable,
-    // someone can add attchments on this
-    HasAttachments,
+    // someone can add attachment on this
+    Attachable,
```

Comment:
attachable means something else though ... like you can _attach this to_ something and that this can have attachments... that's why we had the different name ...

how about ...
```suggestion
    Attachmentable,
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1208/commits/e2b3ca95232d7fedd48a158f1c42657de5d9e794